lib/chewy/fields/base.rb

```diff
@@ -53,6 +55,18 @@ def compose(*objects)
 
     private
 
+      def field_with_ignore_blank_flag?(result)
+        @options[:ignore_blank] == true && field_with_empty_value?(result)
+      end
+
+      def geo_point_field_without_ignore_blank_flag?(result)
+        !@options[:ignore_blank] && @options[:type] == :geo_point && field_with_empty_value?(result)
+      end
```

Comment:
here we should take a look at 2 things:

#### First:

Let's assume that someone really wants raise an error when field with geo_point type is blank (because it was supposed to be always filled or any other reason). If they explicitly pass option `ignore_blank: false` we should expect him to receive an error if field is blank, don't you think?

This case also should be tested, and we assume that the expected behavior is to raise an error.

#### Second:

Instead of checking for `@options[:type] == :geo_point` we should check `@options[:type].to_sym == :geo_point`
to allow users to pass types as string.

Reply:
Thank you!

About the 1st comment: 
I added the spec for 
`field :location, type: :geo_point, ignore_blank: false`.

And
{'id' => 1, 'name' => 'City1', 'location' => {}},
{'id' => 2, 'name' => 'City2', 'location' => ''}

returns

{'id' => 1, 'location' => {}, 'name' => 'City1'},
{'id' => 2, 'location' => '', 'name' => 'City2'}

and does not return an error